modules/apps/client-extension/client-extension-type-impl/src/test/java/com/liferay/client/extension/type/internal/factory/ThemeCSSCETImplFactoryImplTest.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.client.extension.type.internal.factory;
+
+import com.liferay.client.extension.exception.ClientExtensionEntryTypeSettingsException;
+import com.liferay.client.extension.type.ThemeCSSCET;
+import com.liferay.client.extension.type.internal.ThemeCSSCETImpl;
+import com.liferay.portal.json.JSONFactoryImpl;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.util.UnicodePropertiesBuilder;
+import com.liferay.portal.test.rule.FeatureFlags;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * @author Thiago Buarque
+ */
+@FeatureFlags("LPD-10773")
+public class ThemeCSSCETImplFactoryImplTest {
+
```

Comment:
why not putting this in `client-extension-type-test`?

Reply:
Creating a integration test for a class that does not require components to work seems unnecessary. Since unit test is faster, it seems more reasonable